translate/src/modules/interactivetour/components/InteractiveTour.tsx

```diff
@@ -68,7 +68,7 @@ export function InteractiveTour(): React.ReactElement | null {
           </Localized>
           <Localized id='interactivetour-InteractiveTour--main-toolbar-footer'>
             <p>
-              On the right hand side, logged in users can access notifications
+              On the right hand side, logged-in users can access notifications
```

Comment:
I missed that earlier, sorry:

We [localize](https://github.com/mozilla/pontoon/tree/master/translate#localization) the translate view, so you'll also need to update the corresponding localization file.

Fix that and we're ready to merge!